decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class Decorator < Nameable
```

Comment:
[OPTIONAL]
- [ ] Great job on implementing the Decorator class correctly, however adding the nameable in a `setter` would make it greater.
 ```suggestion
class Decorator < Nameable
attr_accessor :nameable
```

Reply:
Thanks